Gemfile

```diff
@@ -25,6 +25,7 @@ group :development, :test do
   gem 'capybara'
   gem 'selenium-webdriver'
   gem 'sqlite3'
+  gem 'dotenv'
```

Comment:
`heroku local` already loads stuff from `.env`. What are we gaining with `dotenv` gem? I'm not opposed to this if you find it better, but we will need to be careful as it can get weird with test envs if we aren't careful.